src/sources/tripadvisor/pois/convert.rs

```diff
@@ -15,8 +15,14 @@ use super::models::Property;
 use crate::langs::COUNTRIES_LANGS;
 use crate::sources::tripadvisor::build_id;
 
-/// Required review count to get the maximal weight of 1.
-const MAX_REVIEW_COUNT: u64 = 1000;
+/// Maximal rating possible
+const MAX_RATING: f64 = 5.;
+
+/// Required review count to get to a weight of 1 with a maximal rating
+const HIGH_REVIEW_COUNT: f64 = 5_000.;
+
+/// Constant offset of the POI to make it a bit easier to match tripadvisor POIs.
+const WEIGHT_BOOST: f64 = 0.3;
```

Comment:
Yeah, I was also thinking about that on my way to work: it would also allow to change a weight without requiring ~3 PR to change the image, pin a version and deploy it.